scripts/packages/postremove.sh

```diff
@@ -7,13 +7,23 @@ if [ "$ID" = "freebsd" ]; then
     echo "Stop and remove nginx-agent service"
     service nginx-agent onestop >/dev/null 2>&1 || true
     sysrc -x nginx_agent_enable >/dev/null 2>&1 || true
+    echo "Removing etc directory"
```

Comment:
Could you update the echos here to have the full path so that its clear what directories are being removed.
i.e. `echo "Removing /usr/local/etc/nginx-agent directory"`